project_task_required_project/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Ángel García de la Chica Herrera <angel.garcia@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    is_project_required = fields.Boolean(default=False, string="Project Required")
```

Comment:
```suggestion
    is_project_required = fields.Boolean(default=False, string="Project required on tasks")
```

Reply:
Small improvement proposition to have a better understanding of the parameter in UI